docs/catalog/cloud/scale-infrastructure-with-user-load.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: franziska-nttdata-hesselfeld
+published_date: TBD
+category: cloud
+tags: 
+ - compute
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Scale infrastructure with user load #
+
+## Description
+
+This pattern involves reviewing workloads for predictable patterns and identifying periods of low or no activity where resources can be scaled back.
+
+## Solution
+
+Scaling back resources during periods of low utilisation eliminates excess capacity and improves efficiency because a unit running at a high utilisation produces energy more efficiently. In addition, reducing the total number of resources used also reduces the amount of embodied carbon.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, scale infrastructure with user laod, will impact two parts:
+
+- `M`: Can reduce the embodied emissions if the application after the optimization uses less hardware overall
+- `E`: Optimizing the application will lead to less electricity consumed over time
+
+## Assumptions
+
+- Optimization needs proficiency in the tools and time to be implemented.
```

Comment:
Are there any specific tools that need proficiency for this?

Reply:
Good question! After thinking about it, there is no specific tool which need proficiency for this. So I will delete it. Is it ok, to have a blank assumptions list?

Reply:
Yes, blank lists are generally OK. You may uncover some during the SME Review section as well.